power_grid_model_c/power_grid_model/include/power_grid_model/main_model.hpp

```diff
@@ -1181,6 +1184,20 @@ class MainModelImpl<ExtraRetrievableTypes<ExtraRetrievableType...>, ComponentLis
             y_bus_vec.reserve(n_math_solvers_);
             auto math_params = get_math_param<sym>();
 
+            IdxVector branch_param_in_seq_map{};
+            IdxVector shunt_param_in_seq_map{};
```

Comment:
not a fundamental type or struct without default values for its members and default constructor, so you can omit `{}`

```suggestion
            IdxVector branch_param_in_seq_map;
            IdxVector shunt_param_in_seq_map;
```

Reply:
Resolved in latest commit.